docs/admin/engines/settings.rst

```diff
@@ -219,6 +219,25 @@ Communication with search engines.
 ``max_redirects`` :
   30 by default. Maximum redirect before it is an error.
 
+``main_categories:``
+--------------------
+
+A list of the categories that are displayed as tabs in the user interface.
+
+.. code-block:: yaml
+
+  main_categories:
+    - general
+    - images
+    - videos
+    - news
+    - map
+    - music
+    - it
+    - science
+    - files
+    - social media
+
```

Comment:
> I documented `main_settings`

Can you please add a documentation of main- & secondary categories here, like you gave in the comments above https://github.com/searxng/searxng/pull/634#issuecomment-1001010436

I also miss a description for the users / https://github.com/searxng/searxng/pull/634#issuecomment-1001011772 ...

>> they can now search wikis with `!wikis`, repositories with `!repositories`, etc.

> One more, I think we do need a description for the users https://searxng.github.io/searxng/user/search_syntax.html